app/uk/gov/hmrc/exports/services/mapping/CachingMappingHelper.scala

```diff
@@ -24,10 +24,10 @@ class CachingMappingHelper {
   val defaultMeasureCode = "KGM"
 
   def commodityFromExportItem(exportItem: ExportItem): Commodity = {
-    def removeCarriageReturns(description: String): String = description.replaceAll("(\\r\\n)|\\r|\\n", " ")
+    def removeCarriageReturns(description: Option[String]): Option[String] = description.map(_.replaceAll("(\\r\\n)|\\r|\\n", " "))
 
     Commodity(
```

Comment:
Q.  If the resulting Commodity now ends up containing nothing (because all the inputs are now optional) do we need to change this method to return Option[Commodity] and ignore this part of in the XML payload. 
I did something similar recently when changing the CommodityMeasure code in the BE.

Reply:
Ahhh.... Really good point!

Reply:
This has been addressed!